templates/php/base/params.twig

```diff
@@ -16,6 +20,9 @@
         {%~ for parameter in method.parameters.header %}
         $apiHeaders['{{ parameter.name }}'] = ${{ parameter.name | caseCamel | escapeKeyword }};
         {%~ endfor %}
+        {%~ if method.name | lower == "createexecution" %}
+        $apiHeaders['accept'] = 'multipart/form-data';
+        {%~ endif %}
```

Comment:
I think we should move this code to the client class, inside  the case for multi part 

![Screenshot 2024-08-30 at 11 40 04 AM](https://github.com/user-attachments/assets/06ac017e-a1eb-4e83-9d2f-629cca41ecff)

Reply:
Or we should pass some sort of an attribute from the specs to make this check easier